plugins/modules/boot.py

```diff
@@ -384,15 +384,22 @@ def main():
                 option = module.params[option_name][option_key]
                 if option is None or option == []:
                     continue
-                data[data_key] = option
+
+                if isinstance(option, list):
+                    data[data_key+'[]'] = option
```

Comment:
What are the `[]` for? The API docs do not mention that.

Reply:
I tried this out, and cannot figure out why you add the `[]`. It does work fine without, and that's also what the API docs document. Please remove this.